inspirehep/records/api/literature.py

```diff
@@ -6,19 +6,26 @@
 # the terms of the MIT License; see LICENSE file for more details.
 
 """INSPIRE module that adds more fun to the platform."""
-
+import json
+import logging
 
 from inspire_schemas.builders import LiteratureBuilder
 
-from ...pidstore.api import PidStoreLiterature
+from inspirehep.pidstore.api import PidStoreLiterature
+
 from .base import InspireRecord
 
+logger = logging.getLogger(__name__)
+
 
 class LiteratureRecord(InspireRecord):
     """Literature Record."""
 
     pid_type = "lit"
 
+    es_serializer = "LiteratureESEnhancementV1"
+    ui_serializer = "LiteratureMetadataSchemaV1"
```

Comment:
string reference? can we just pass the classes?

Reply:
Unfortunately not. Circular dependencies ;)